docs/conf.py

```diff
@@ -29,10 +29,6 @@
 # so we just want to display the major and minor version in the tab title of the browser.
 release = ".".join(rb.__version__.split(".")[0:2])
```

Comment:
I think with the changes we should now follow the convention laid out here: https://www.sphinx-doc.org/en/master/usage/configuration.html
So `release` is the full project version with bugfix/prerelease tag, and `version` is just the major and minor digits.
```suggestion
release = rb.__version__
```